_opensearch/snapshots/snapshot-management.md

```diff
@@ -0,0 +1,78 @@
+
+When you create an SM policy, its document ID is given the name `<policy_name>-sm-policy`. Because of this, SM policies have to obey the following rules:
+
+- SM policies must have unique names. 
+
+- You cannot update the policy name after its creation. 
+
+SM-created snapshots have names in the format `<policy_name>-<date>-<random number>`. Two snapshots created by different policies at the same time always have different names because of the `<policy_name>` prefix. To avoid name collisions within the same policy, each snapshot's name contains a random string suffix.
+
+Each policy has associated metadata that stores the policy status. Snapshot management saves SM policies and metadata in the system index, and reads them from the system index. Thus, snapshot management depends on the OpenSearch cluster's indexing and searching functions. The policy's metadata keeps information about the latest creation and deletion only. The metadata is read before running every scheduled job, so that SM can continue execution from the previous job's state. You can view the metadata using the [explain API]({{site.url}}{{site.baseurl}}/opensearch/snapshots/sm-api#explain).
+
+An SM schedule is a custom [cron]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron) expression. It consists of two parts: a creation schedule, and a deletion schedule. You must set up a schedule that specifies the frequency and timing of taking snapshots. Optionally, you can set up a separate schedule for deleting snapshots.
+
+An SM configuration includes the indices and repository for the snapshots, and supports all parameters you can define when [creating a snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots) using the API. Additionally, you can specify the format and time zone for the date used in the snapshot's name.
+
+
+## Performance 
+
+One snapshot can contain as many indices as there are in the cluster. We expect at most dozens of SM policies in one cluster, but a snapshot repository can safely scale to thousands of snapshots. However, to manage its metadata, a large repository requires more memory on the cluster manager node. 
+
+Snapshot management depends on the job scheduler plugin to schedule a job that is run periodically. Each SM policy corresponds to one SM scheduled job. The scheduled job is lightweight, so the burden of SM depends on the snapshot creation frequency and the burden of running the snapshot operation itself. 
+
+## Concurrency 
+
+An SM policy does not support concurrent snapshot operations, since too many such operations may degrade the cluster. Snapshot operations (creation or deletion) are performed asynchronously: SM does not start a new operation until the previous operation finishes.
```

Comment:
Snapshot operations (creation or deletion) are performed asynchronously and SM does not start a new operation until the previous asynchronous operation finishes.